### DISCUSSION TITLE
Dependency errors when swapping out three js for our own fork of three js

### DISCUSSION THREAD

**Discussion by honeyimholm** [mozilla/hubs#2711] [General Questions & Discussion] [answered]
Hi all I've been trying to integrate a custom fork of three js but have continually run into peer dependency issues after changing the package.json and running npm install. I tried making a fork of aframe as well to change its three js github link in the package,json and then changing the hubs repo's aframe github link to the custom fork.  With this we're now getting this:
 
`npm WARN notsup Unsupported engine for aframe@1.0.3: wanted: {"node":">= 4.6.0","npm":"^2.15.9"} (current: {"node":"12.16.1","npm":"6.13.4"})`

Before the aframe change we were seeing:

`npm WARN aframe@1.0.3 requires a peer of three@github:mozillareality/three.js#hubs/master but none is installed. You must install peer dependencies yourself.`

Even though these are warnings npm build deploy fails 

Any suggestions?

**Comment by robertlong** [comment, marked as answer]
It's hard to say what's going on here without more info. Are you basing your custom fork of three off of ours? I don't think we can provide much support here unless your custom fork is our hubs/master with a few modifications.
